test/expected/bgw_launcher.out

```diff
@@ -295,6 +295,38 @@ SELECT wait_worker_counts(1,0,0,0);
  t
 (1 row)
 
+/* Test that background workers are stopped with DROP OWNED */
+ALTER ROLE :ROLE_DEFAULT_PERM_USER WITH SUPERUSER;
+\c single_2  :ROLE_DEFAULT_PERM_USER
+SET client_min_messages = ERROR;
+CREATE EXTENSION timescaledb CASCADE;
+RESET client_min_messages;
+/* Make sure there is 1 launcher and 1 bgw in single_2 */
+SELECT wait_worker_counts(launcher_ct=>1, scheduler1_ct=> 0, scheduler2_ct=>1, template1_ct=>0);
+ wait_worker_counts 
+--------------------
+ t
+(1 row)
+
+/* drop a non-owner of the extension results in no change to worker counts */
```

Comment:
The test can't really tell the difference between case a) where we drop owned and the worker isn't even restarted, which is the desired behavior and b) we drop owned and the worker is restarted and it stays alive because the extension wasn't dropped. I'm not sure that it matters here though. The way to do it would be use a similar thing to test that the worker wasn't restarted (ie the backend start time remains constant).